src/representation_theory/character.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright (c) 2022 Antoine Labelle. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Labelle
+-/
+import representation_theory.fdRep
+import linear_algebra.trace
+import representation_theory.basic
+
+/-!
+# Characters of representations
+
+This file introduces characters of representation and proves basic lemmas about how characters
+behave under various operations on representations.
+
+# TODO
+* Once we have the monoidal closed structure on `fdRep k G` and a better API for the rigid
+structure, `char_dual` and `char_lin_hom` should probably be stated in terms of `Vᘁ` and `ihom V W`.
+-/
+
+noncomputable theory
+
+universes u
+
+open linear_map category_theory.monoidal_category representation
+
+variables {k G : Type u} [field k]
+
+namespace fdRep
+
+section monoid
+
+variables [monoid G] (V W : fdRep k G)
+
```

Comment:
```suggestion
variables [monoid G] (V W : fdRep k G)

```
Just one comment here: it think it would be nice to put explicit arguments like `V` and `W`  directly in the statements of the relevant lemmas. It can get a bit annoying to have to hunt around for variables declarations several lines up to figure out how to use a lemma we're looking at.